mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -239,7 +239,7 @@ The following arguments are supported:
 
 * `database_version` - (Required) The MySQL, PostgreSQL or
 SQL Server version to use. Supported values include `MYSQL_5_6`,
-`MYSQL_5_7`, `MYSQL_8_0`, `POSTGRES_9_6`,`POSTGRES_10`, `POSTGRES_11`,
+`MYSQL_5_7`, `MYSQL_8_0`, `MYSQL_8_4`, `POSTGRES_9_6`,`POSTGRES_10`, `POSTGRES_11`,
```

Comment:
The `MYSQL_8_4` is only added to the documentation. Does it need to be added to somewhere else to be supported?

Reply:
Updated PR title